application/frontend/src/scaffolding/Header/Header.tsx

```diff
@@ -15,7 +15,24 @@ const getLinks = (): { to: string; name: string }[] => [
 ];
 
 export const Header = () => {
-  const { params, url, showHeader } = useLocationFromOutsideRoute();
+const HandleDoFilter = () => {
+    currentUrlParams.set("applyFilters", "true");
+    history.push(window.location.pathname + "?" + currentUrlParams.toString());
+  }
+
+  const ClearFilter = () => {
```

Comment:
Normally functions are lowerCamelCase in js.
When you have `export const SomeComponent`, you use PascalCase to denote that is a React component.

TL;DR: naming convention is lowerCamelCase for functions , PascalCase for React components